### DISCUSSION TITLE
:bug:  Meaning of "IOException: IO Error: Read error for HTTP GET Range to" :grey_question:

### DISCUSSION THREAD

**Discussion by adriens** [duckdb/duckdb#6870] [Q&A]
Sometimes, when I run the following sql : 

```sql
select distinct platform
from
read_json_auto('https://dist.tea.xyz/bottles.json')
order by platform;
```

I get the following error : 

```---------------------------------------------------------------------------
IOException                               Traceback (most recent call last)
/tmp/ipykernel_28/2190390420.py in <module>
      3 select distinct platform
      4 from
----> 5 read_json_auto('https://dist.tea.xyz/bottles.json');""")
      6 
      7 con.execute("""create or replace table architecture (id varchar not null primary key);""")

IOException: IO Error: Read error for HTTP GET Range to 'https://dist.tea.xyz/bottles.json'
```

![image](https://user-images.githubusercontent.com/5235127/227801477-7aef3e5c-82e0-488d-b290-6117b0e23ee6.png)

ANy idea what's going wrong ?... and where ?

Thank you in advance for your help.

**Comment by carlopi** [comment]
I do reproduce, and I believe it's a problem similar to #6497, will investigate further.

**Reply by carlopi** [reply]
Actually, unsure, as in this case it seems that it might be actually timing out (it takes in the order of seconds also with curl / wget do download).

**Reply by adriens** [reply]
indeed, yes, That's the conclusion I also had :+1:

**Reply by adriens** [reply]
:point_right: BTW, yes, it's gzipped from cloudfront
